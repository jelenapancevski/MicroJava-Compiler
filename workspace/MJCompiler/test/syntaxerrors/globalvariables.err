ERROR 19:43:11,662 - 
Sintaksna greska na liniji 2
ERROR 19:43:11,664 - Greska kod deklaracije promenljive.
ERROR 19:43:11,666 - 
Sintaksna greska na liniji 3
ERROR 19:43:11,667 - Greska kod deklaracije promenljive.
ERROR 19:43:11,668 - 
Sintaksna greska na liniji 5
ERROR 19:43:11,669 - Greska kod deklaracije promenljive.
ERROR 19:43:11,669 - 
Sintaksna greska na liniji 6
ERROR 19:43:11,670 - Greska kod deklaracije promenljive.
ERROR 19:43:11,670 - 
Sintaksna greska na liniji 7
ERROR 19:43:11,671 - Greska kod deklaracije promenljive.
ERROR 19:43:11,674 - 
Sintaksna greska na liniji 9
ERROR 19:43:11,675 - Greska kod deklaracije promenljive.
ERROR 19:43:11,699 - Parsiranje koda je neuspesno
