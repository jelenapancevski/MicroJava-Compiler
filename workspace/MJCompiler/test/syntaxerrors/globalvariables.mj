program globalvariables 
	char slovoa, char char slovob ;
	int x, int int, y;
	
	int a[], + abc, length, b[] ;
	bool a a, boll;
	bool  d f g;
{
	void main() int az;; int bz;{
	slovoa= 'a';
	x = 1;
	y = x+5;
	length = 10;
	a = new int [length];
	b = new int [y];
	
	}
}