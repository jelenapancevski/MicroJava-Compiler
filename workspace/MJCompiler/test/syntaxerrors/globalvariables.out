Error: Could not find or load main class rs.ac.bg.etf.pp1.Compiler
Caused by: java.lang.ClassNotFoundException: rs.ac.bg.etf.pp1.Compiler
